### ISSUE TITLE
pgclimb should respect the usual PostgreSQL variables and .pgpass file

### ISSUE THREAD

**Issue by alexeyklyukin** [lukasmartinelli/pgclimb#3]
http://www.postgresql.org/docs/9.5/static/libpq-envars.html

I find it rather unsafe to either specify the password from the command-line, or use the environment variable.

**Comment by lukasmartinelli** [comment]
> I find it rather unsafe to either specify the password from the command-line, or use the environment variable.

Yes I could support the standard env vars that `psql` already is using.
With unsafe you mean you would prefer pgpass file support?

**Comment by alexeyklyukin** [comment]
Yes, absolutely. pgpass normally has very restrictive access privileges (typically 600), so it's less a risk than exposing your password in the environment variable or passing it via a command-line option.

**Comment by machytkafitanalytics** [comment]
+1 for .pgpass

**Comment by nyurik** [comment]
+1 for standard env vars - makes automation easier too.
